src/Microsoft.ComponentDetection.Detectors/pip/Contracts/PipInstallationMetadata.cs

```diff
@@ -0,0 +1,79 @@
+namespace Microsoft.ComponentDetection.Detectors.Pip;
+
+using Newtonsoft.Json;
+
+/// <summary>
+/// Metadata for a pip component being installed. See https://packaging.python.org/en/latest/specifications/core-metadata/.
+/// Some fields are not collected here because they are not needed for dependency graph construction.
+/// </summary>
+public class PipInstallationMetadata
+{
+    /// <summary>
+    /// Version of the file format; legal values are “1.0”, “1.1”, “1.2”, “2.1”, “2.2”, and “2.3”
```

Comment:
nit: are these smart quotes?

```suggestion
    /// Version of the file format; legal values are "1.0", "1.1", "1.2", "2.1", "2.2", and "2.3"
```

Reply:
Yeah it looks like they were 😆 Copied them directly from the documentation.